lib/base/utility.hpp

```diff
@@ -109,6 +109,7 @@ class Utility
 	static tm LocalTime(time_t ts);
 
 	static bool PathExists(const String& path);
+	static time_t GetFileCreationTime(const String& path);
```

Comment:
The addition of this one shall be an extra commit with a proper message.

Reply:
with a proper commit message in the same PR or also an own PR?

Reply:
Same PR.